packages/node_modules/@webex/internal-plugin-encryption/src/encryption.js

```diff
@@ -113,17 +101,36 @@ const Encryption = WebexPlugin.extend({
       }
     })
       .then((res) => {
-        const url = res.body.endpoints[scr.loc];
+        const url = scr.loc;
 
-        if (!url) {
-          this.logger.warn('encryption: could not determine download url for `scr.loc`; attempting to download `scr.loc` directly');
+        return _this2._callDownloadUrl(url, scr, res);
+      });
+  },
 
-          return scr.loc;
-        }
-        this.logger.info('encryption: retrieved download url for encrypted file');
+  _callDownloadUrl(url, scr, result) {
+    const _this2 = this;
```

Comment:
I'm not too sure why `this` is being reinstated here. Typically, we should avoid this type of declaration wherever possible. In cases in which `this` needs to be a different object in memory, it can be associated using `method.bind(myObj);` where `myObj` would be aliased as `this`.

Alternatively, since these appear to be class methods, simply removing these lines and utilizing `this` may be a solution.

Also note, Javascript makes namespace assignments mutable for non-primitive data, so both `this` and `_this2` point to the same space in memory, making them identical in every way even after data has been changed on either namespace.